server/app/api/v3.py

```diff
@@ -64,18 +68,30 @@ def __init__(self, s3_uri_root):
 
 class SchemaAPI(S3URIResource):
```

Comment:
SchemaAPI: caching switched to no-store to accommodate per-user annotation schema. Please confirm expected CDN/proxy behavior and that this won’t regress load times.